vms/rpcchainvm/vm.go

```diff
@@ -39,20 +39,15 @@ func Serve(ctx context.Context, vm block.ChainVM, opts ...grpcutils.ServerOption
 	server := newVMServer(vm, opts...)
 	go func(ctx context.Context) {
 		defer func() {
+			signal.Stop(signals)
 			server.GracefulStop()
```

Comment:
With this PR, we don't except this to be hit right?

Reply:
that is the only downside of this approach is we don't allow the gRPC server to gracefully shutdown.

Reply:
correct - this would only be hit if the context finishes (which isn't part of the expected shutdown flow)